controllers/existing.go

```diff
@@ -0,0 +1,32 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-logr/logr"
+	appsv1 "k8s.io/api/apps/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	kedaCoreLabels = map[string]string{"app": "keda-operator", "app.kubernetes.io/name": "keda-operator"}
+)
+
+func HasExistingKedaInstallation(c client.Client, logger logr.Logger) (bool, error) {
```

Comment:
the imported package is `github.com/kyma-project/keda-manager/keda`

Reply:
> Wouldn't hat be confusing? This is the operator that's supposed to deploy "our" keda deployment. It's not clear which `keda.IsInstalled` in this case.. ours or the upstream.

I think for the `HasExistingKedaInstallation` we have the same case. After reading this name I can't tell if it's about `our` or `upstream`. But the good argument for moving this method and renaming it is that we can create a dedicated package with helpers for the Keda domain (called `keda`) and we can locate this method inside it because probably in the future we will have more similar functions. I think the name `IsInstalled` is good enough but if you think it is too general we can rename it to add more details or just simply comment on this function (it's good practice as well).